### DISCUSSION TITLE
Schedule Target Temperature

### DISCUSSION THREAD

**Discussion by jamesonuk** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#445] [Ideas]
I currently use my hot tub primarily over the weekend.   Would be really useful if I was able to say I want the tub to be at  38°C at 19:00 on Friday at 18:00 and stay at that temperature until Sunday at 18:00  (at which point I will turn the heater off)

Then make use of the time to heat estimates and work out when to turn the heater on to hit the target temperature at the target time.

So rather than saying when you want the heater to come on, say when you want the tub to be up to temperature and let the code determine when that means the heater needs to go on.;

**Comment by visualapproach** [comment]
@jamesonuk that feature is already implemented in the not stable/not documented v4 branch. I'll focus more on trying to get that version stable for the moment, than adding it to v3. Thanks for your good suggestion!
